src/analysis/processing/qgsalgorithmdifference.cpp

```diff
@@ -83,6 +83,10 @@ void QgsDifferenceAlgorithm::initAlgorithm( const QVariantMap & )
   addParameter( new QgsProcessingParameterFeatureSource( QStringLiteral( "INPUT" ), QObject::tr( "Input layer" ) ) );
   addParameter( new QgsProcessingParameterFeatureSource( QStringLiteral( "OVERLAY" ), QObject::tr( "Overlay layer" ) ) );
   addParameter( new QgsProcessingParameterFeatureSink( QStringLiteral( "OUTPUT" ), QObject::tr( "Difference" ) ) );
+  std::unique_ptr< QgsProcessingParameterNumber > gridSize = std::make_unique< QgsProcessingParameterNumber >( QStringLiteral( "GRIDSIZE" ),
```

Comment:
These parameters should be hidden for geos <3.9 build, and a QgsProcessingException raised if they are specified and non zero in the processAlgorithm stages on < 3.9 builds.

Reply:
Actually you can ignore this now -- we've just bumped the minimum GEOS to 3.9 so now we can be sure it's always available

Reply:
Nice!